### PR TITLE
updated README to include extra how to run step

### PR DESCRIPTION
**Pull request by skylarbarrera**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,13 +26,15 @@ If you are new to React Native, this is a helpful introduction: https://facebook
...
 4. Run `yarn nodeify`.
 
-5. Install required Pods by running `yarn install-pods`.
+5. Run `yarn install-bundle`.
 
-6. Run `yarn ios` to build the project for XCode.
+6. Install required Pods by running `yarn install-pods`.
 
-7. Open `rainbow-wallet/ios/RainbowWallet.xcworkspace`.
+7. Run `yarn ios` to build the project for XCode.
```
Comment (osdnk) [inline README.md:33]: Can we drop this? it seems to be not needed!

Also, I think it's worth mentioning that running bundler is important

File: README.md [hunk 2]
```diff
@@ -26,13 +26,15 @@ If you are new to React Native, this is a helpful introduction: https://facebook
 
 4. Run `yarn nodeify`.
```
Comment (osdnk) [inline README.md:27]: i think it's no longer needed. 

Also, the part about `nvm` should be optional. Personally, i don't use it.

**Review by osdnk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (skylarbarrera) [reply README.md:33]: Will clear up the other inconsistencies after I meet with Bruno

File: README.md (hunk 2 above)
Comment (brunobar79) [reply README.md:27]: Then we need to specify the node version, there are certain things like Ganache that doesn't work with node 14.
Node 10 & 12 works fine.

**Comment by osdnk** [comment]
Looks cool. After fixing press "Resolve conversion"

**Pull request merged**